pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/TestKitUtil.java

```diff
@@ -170,86 +168,83 @@ private static void delete(Connection conn, String tableName, String primaryColu
             throw new RuntimeException("Error deleting ", ex);
         }
     }
-    private static void delete(Connection conn, String tableName, String primaryColumn, List<String> ids) {
-        for(String id: ids){
+
+    private void delete(Connection conn, String tableName, String primaryColumn, List<Integer> ids) {
+        for (int id: ids) {
             delete(conn, tableName, primaryColumn, id);
         }
-
     }
 
-    public static void deleteKit(String ddpKitRequestId) {
+    public void deleteKit(String ddpKitRequestId) {
         SimpleResult results = inTransaction((conn) -> {
-            List<String> dsmKitRequestId = new ArrayList<>();
+            List<Integer> dsmKitRequestId = new ArrayList<>();
             try (PreparedStatement stmt = conn.prepareStatement(SELECT_DSM_KIT_REQUEST_ID)) {
-                stmt.setString(1, ddpKitRequestId.concat("\\_%"));
```

Comment:
@pegahtah when I set a breakpoint here, it became clear that this pattern match resulted in zero rows being deleted, instead of the given `ddpRequestId`, which then lead to downstream errors during cleanup methods because ddp_instance rows could not be deleted because of the fk from ddp_kit_request.  Can you confirm that this pattern match is not needed?  Once I changed the delete to use the request id exactly, tests started passing.